x/marker/keeper/keeper.go

```diff
@@ -176,3 +184,8 @@ func (k Keeper) IterateMarkers(ctx sdk.Context, cb func(marker types.MarkerAccou
 func (k Keeper) GetEscrow(ctx sdk.Context, marker types.MarkerAccountI) sdk.Coins {
 	return k.bankKeeper.GetAllBalances(ctx, marker.GetAddress())
 }
+
+// GetAuthority is signer of the proposal
+func (k Keeper) GetAuthority() string {
```

Comment:
This is a pretty simple function, but it would be nice to have a test for it. Maybe something like what we have in the name module? The following code is what we have in the name module, and may need to be adjusted slightly:

```
func (s *KeeperTestSuite) TestGetAuthority() {
	s.Run("has correct authority", func() {
		authority := s.app.NameKeeper.GetAuthority()
		s.Require().Equal("cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn", authority)
	})
}
```

Reply:
Besides that, everything else looks really good :).